.github/workflows/image.yml

```diff
@@ -1,19 +1,14 @@
 name: image
 
 on:
-  push:
-    branches: [ main ]
-    paths-ignore:
-      - 'doc/**'
-      - 'enhancements/**'
-      - '*.md'
+  workflow_call:
 
```

Comment:
IIRC, only image.yml here is active workflow, image_base.yml and image_pr.yml are inactive ones.
Please confirm with @rootfs, since your current PR is a workflow cleanup and refactoring.

Reply:
+ @jichenjc here in loop.
- for image_pr.yml, which is used to build PR level image, as @rootfs request in the history. For our feature branch test usage, hence I suppose we need to keep it, in case of we have a big PR for example libebpf or others which need test locally. as far as I remember, we have this feature to build PR branch's image and upload to container reg, then cross validation.
- for image_base.yml, @jichenjc , @rootfs , as github agent has it's own linux header makes linux-header fails install in a ubi base image during image build.( there is no rpm package for github agent's linux header as they are ubuntu with deb package management system) Hence, any idea for how we build and daily build base image to reduce any CVE at OS level? or some yum package as linux-header?